client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -95,55 +97,137 @@ const ArtistHeaderSection: React.FC<{ artist: Artist; isManage?: boolean }> = ({
-                  {!isManage && <FollowArtist artistId={artist.id} />}
-                  {!isManage && user?.id === artist.userId && (
-                    <Link to={`/manage/artists/${artist.id}`}>
-                      <Button
-                        compact
-                        transparent
-                        type="button"
-                        startIcon={<FaPen />}
-                      >
-                        {t("edit")}
-                      </Button>
-                    </Link>
-                  )}
+                <div
+                  className={css`
+                    ${artistAvatar ? "min-height: 50px;" : ""}
+                    display: flex;
+                    align-items: center;
+                    justify-content: space-between;
+                    width: 100%;
+                  `}
+                >
+                  <div>
+                    <H1
+                      className={css`
+                        @media screen and (max-width: ${bp.medium}px) {
+                          ${!artistAvatar ? "font-size: 1.5rem !important" : ""}
+                        }
+                      `}
+                    >
+                      {artist.name}
+                    </H1>
+                  </div>
+
+                  <div
+                    className={css`
+                      text-align: right;
+                      display: flex;
+                      flex-direction: column;
+                      padding-left: 1rem;
+                      @media screen and (max-width: ${bp.medium}px) {
+                        padding-left: 0.3rem;
+                      }
+                    `}
+                  >
+                    {!isManage && <FollowArtist artistId={artist.id} />}
+                    {!isManage && user?.id === artist.userId && (
+                      <Link to={`/manage/artists/${artist.id}`}>
+                        <Button
+                          compact
+                          transparent
+                          type="button"
+                          startIcon={<FaPen />}
+                        >
+                          {t("edit")}
+                        </Button>
+                      </Link>
+                    )}
+                  </div>
                 </div>
               </HeaderDiv>
-              <MarkdownContent content={artist.bio} />
+              <div
+                className={css`
+                  ${artistAvatar ? "display: none;" : ""}
+                  ${!artistAvatar ? "padding-bottom: .2rem;" : ""}
+                `}
+              >
+                <MarkdownContent
+                  content={artist.bio}
+                  className={css`
+                    ${!artistAvatar ? "margin-bottom: .7rem !important" : ""}
+                  `}
+                />
+              </div>
             </div>
           </div>
+          <div
+            className={css`
+              ${!artistAvatar ? "display: none;" : ""}
```

Comment:
Ditto for ternary clause here

Reply:
I think I fixed everything except that last one? What would I need to do here to make the code better? It's the only I found to make sure this didn't display when there's no avatar.

Reply:
This and the next line could be turned into one statement